### PR TITLE
Fix#1340 Fix VaButton icon color

### PR DESCRIPTION
**Pull request by RVitaly1978**
## Description
close: #1340 

Changes:
- [x] - fix icon color according to `text-color` prop
- [x] - update demo and docs examples

image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-button/VaButton.vue [hunk 1]
```diff
@@ -160,6 +162,10 @@ export default class VaButton extends mixins(
     return this.$slots.default
   }
 
+  get hasOneIcon () {
+    return (this.iconRight && !this.icon) || (!this.iconRight && this.icon) || false
```
Comment (m0ksem) [inline packages/ui/src/components/va-button/VaButton.vue:166]:
```suggestion
    return Boolean((this.iconRight && !this.icon) || (!this.iconRight && this.icon))
```

**Review by m0ksem — APPROVED** [review]
Good. A few things can be improved, but it is not required.

File: packages/ui/src/components/va-button/VaButton.vue [hunk 2]
```diff
@@ -29,13 +29,15 @@
...
         :size="size"
+        :style="{ color: textColorComputed }"
         class="va-button__left-icon"
       />
       <slot />
       <va-icon
         v-if="iconRight"
         :name="iconRight"
         :size="size"
+        :style="{ color: textColorComputed }"
```
[the remark this answers is not in the document]
Comment (RVitaly1978) [reply packages/ui/src/components/va-button/VaButton.vue:40]: fixed it

File: packages/ui/src/components/va-button/VaButton.vue (hunk 1 above)
Comment (RVitaly1978) [reply packages/ui/src/components/va-button/VaButton.vue:166]: fixed it

**Pull request merged**